Wording - LightningNetwork, README, UseCase, Wallet (#492)
btcpayserver/btcpayserver-doc: LightningNetwork.md

This guide will show you how to set up your Lightning Network node in BTCPay and guide you through the basics.

> <strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. Do not use Lightning Network for money you cannot afford to lose. There is a high risk of you losing money with the Lightning Network. </strong>

Reviewer: > Do not use Lightning Network for money you cannot afford to lose

This sounds weird to me. @britttttk what do you think?

Reviewer 2: ```suggestion
<strong>Before you proceed, please understand that Lightning Network is still in the experimental stage. There are high risks. Do not use Lightning Network with funds you can't afford to lose.</strong>
```
